.github/workflows/update-readme.yml

```diff
@@ -14,7 +14,7 @@ jobs:
 
       - name: Prepare
         env:
-          NVIM_TAG: v0.6.1
+          NVIM_TAG: stable
```

Comment:
there's also v0.5.1 used once in another workflow

Reply:
nope, not anymore ;)